markwon/src/main/java/ru/noties/markwon/renderer/html2/MarkwonHtmlRenderer.java

```diff
@@ -30,7 +30,7 @@
 
     public abstract void render(
             @NonNull SpannableConfiguration configuration,
-            @NonNull SpannableBuilder builder,
+            @NonNull Spannable spannable,
```

Comment:
Uououo... Now I see why you have re-used the `setSpan` method in `SpannableBuilder`, but wouldn't it be great if we kept it here as it was? On the bright side we could use some methods that we add (about possible arrays of spans for example).

Reply:
I'm not following you here... Do you mean that I should keep passing a `SpannableBuilder` here and then (define and) call a non-inherited `setSpan(...,` instead of `Spannable`'s `setSpan`?

Reply:
Yeah, keep the original method signature